packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/doenet/p.rs

```diff
@@ -1,11 +1,19 @@
 use crate::components::prelude::*;
 
-#[derive(Debug, Default, ComponentNode, RenderedComponentNode, ComponentStateVariables)]
+#[derive(
+    Debug,
+    Default,
+    ComponentNode,
+    RenderedChildren,
```

Comment:
I could name the derive macro `PassThroughChildren`, which would derive an implementation of the `RenderedChildren` trait that gets the children. I feel, though, that it would be confusing and deviating from convention if the name of the derive macro differed from the name of the trait it implements.

I could have the `RenderedChildren` derive macro define to helper attributes to specify different options for the derive behavior. For example: `#[pass_through_children]` and `#[no_rendered_children]`.

Reply:
And, if one didn't specify an attribute, it would pass through the children.